backend/src/controllers/dbController.ts

```diff
@@ -11,10 +14,70 @@ class DBController {
     }
   }
 
-  async getCurTime(_req: Request, res: Response) {
+  async getInsertUserID(req: Request, res: Response) {
```

Comment:
nit: getInsert ungünstiger Name. Siehe auch für alle folgenden. Funktion sollte nie mehr als eine Aktion ausführen

Reply:
Fixed in commit: 6662bb1c0a3996348032e9a5fcd0588fa5273abe